docs/userGuide/includingContents.md

```diff
@@ -62,6 +62,12 @@
         site.json
       some.md (it include a file in site A, <include src="./A/other.md">)
     ```
+    Path guide:

```

Comment:
`You may reference files from external directories using the following prefixes:`